src/corelibs/U2Algorithm/src/util_msa_consensus/MSAConsensusAlgorithmLevitsky.cpp

```diff
@@ -147,6 +150,9 @@ char MSAConsensusAlgorithmLevitsky::getConsensusChar(const MultipleAlignment& ma
     int selectedGlobalFreq = nSeq * ma->getLength();
     int thresholdScore = getThreshold();
     int minFreq = int(float(nSeq) * thresholdScore / 100);
+    if (minFreq == 0) {
+        minFreq = 1;
```

Comment:
This is an incorrect value. Keep the correct one as 'double' and make ```localFreq``` below a double too

Reply:
No I don't agree with that.
The result will be the same, but algorithm logic became hard to read, and harder to calculate. Because for now we just count consensus symbol hits (integer), convert threshold value percent (double) to hits threshold (integer). But you suggest to calculate frequency (double) for every symbol and compare this with threshold value percent (double).
I can suggest another solution if you don't like mine - it will work properly if we fill `localFreqs` with negative values when it initialized. So in that way '0' minimum frequency became valuable.

Reply:
Frequency of  a symbol is always double: in 'ABC' string the frequency of A is 33.333333...%. 
Keep the original (source) data as long as possible and convert it in the last place.